.gitignore

```diff
@@ -1,5 +1,6 @@
 **/node_modules/**
 **/generated/**
+/service-accounts
```

Comment:
is this required? if so lets add `**/service-accounts/**` so that if this name is used anywhere we ignore it